.github/workflows/test-and-demo.yaml

```diff
@@ -27,3 +27,13 @@ jobs:
       run: npm run build:storybook
     - name: Deploy Storybook
       run: npm run chromatic
+    - name: Extract branch name
```

Comment:
Strange that this is necessary. Apparently github provides the branch name for actions that run on pull requests, but not on push.

Reply:
Yep, I learned this the hard way :x